googletest/README.md

```diff
@@ -24,17 +24,20 @@ another project.
 When building Google Test as a standalone project, the typical workflow starts
 with:
 
-    mkdir mybuild       # Create a directory to hold the build output.
-    cd mybuild
-    cmake ${GTEST_DIR}  # Generate native build scripts.
+    git clone https://github.com/google/googletest.git -b release-1.10.0
```

Comment:
I would add here something like "Even if you want to build only one of the components (e.g. just `googletest`), you have to create your build directory in the main directory of the cloned repo. For people who are as thick as me :)

Reply:
I hope this additional revision would reflect what you commented
Please let me know if there are more parts that need to be corrected